### DISCUSSION TITLE
https://www.npmjs.com/support could not submit from.

### DISCUSSION THREAD

**Discussion by shiyiya** [npm/feedback#712] [Website]
https://www.npmjs.com/support could not submit from.

![image](https://user-images.githubusercontent.com/34017352/174529704-fdd8af45-dd8d-439e-958b-23d6ce97cb1f.png)
![image](https://user-images.githubusercontent.com/34017352/174529741-49678302-0ffb-495b-b1ad-134e65c44886.png)

[x] enable cookie
[x] enable js

I tried different networks and browsers and still get the same error.

Actually, I would like to ask about One-Time Password, my domain name forgot to renew and was registered and I can't receive One-Time Password, so I can't login to my npm account (there was no such verification before) what can I do? I would like to get some help, thanks!

![image](https://user-images.githubusercontent.com/34017352/174530193-60897b52-bff3-49c2-9899-7ad63c235bcc.png)

**Comment by TomasHawKing** [comment]
hi，I also ran into this problem, excuse me, did you solve it now?

**Reply by shiyiya** [reply]
> hi，I also ran into this problem, excuse me, did you solve it now?

No, no one seems to care about that at all

**Reply by monishcm** [reply]
Are you not able to submit a support ticket? We have this tested and do not see a problem submitting a support ticket. Can you give more context and steps that is leading to this error? Screenshots etc. will help. 

Thanks.

**Reply by monishcm** [reply]
Loading captcha seems to be an issue, which is usually caused by blocking the required JavaScript files. Please try troubleshooting as mentioned in the following link: https://docs.github.com/en/get-started/using-github/troubleshooting-connectivity-problems#troubleshooting-the-captcha

**Reply by gc-skyscanner** [reply]
Hey @monishcm, did you solve it now? I failed to sign up an account and raise a support ticket. Both of them expose the captcha error. The troubleshooting doc seems not work for me.
